translator.go

```diff
@@ -6,7 +6,7 @@ import (
 	"time"
 
 	"github.com/mmcdole/gofeed/atom"
-	"github.com/mmcdole/gofeed/extensions"
+	ext "github.com/mmcdole/gofeed/extensions"
```

Comment:
Don't think this import path alias is used.

Reply:
it is being used on line 398. 

I don't understand why it specifically needs declared here, but my editor adds it back in automatically when saving. I'll update it with a different editor. Since it wasn't there before it should be fine.